src/store/trees/bootstrap-legacy-global-dependencies.js

```diff
@@ -1,13 +1,14 @@
 import ImportedLang from 'lang.js'
 import ImportedMoment from 'moment'
 import ImportedJquery from 'jquery'
-import {merge} from 'lodash'
-import source from '../../assets/messages.json';
+import {merge, isEmpty} from 'lodash'
 
-export function bootstrapLegacyGlobalDependencies() {
+export function bootstrapLegacyGlobalDependencies(appConfig = null, builderConfig = null) {
   // initialize configuration sources
-  const __APP__ = require('../../../app.config')
-  const __CONTEXT__ = require('../../../builder.config')
+  const __APP__ = !isEmpty(appConfig) ? appConfig : require('../../../app.config')
+  const __CONTEXT__ = !isEmpty(builderConfig) ? builderConfig : require('../../../builder.config')
+  const source = appConfig.i18n ? appConfig.i18n : require('../../assets/messages.json')
```

Comment:
This logic changes slightly: `appConfig` may be `null` here. A couple possible options: either use lodash's `get(appConfig, 'i18n')` or default the param to an empty dictionary `{}` in the function signature.

Reply:
Ah yes, meant to change it back to `{}` defaults